utilities/validations.py

```diff
@@ -107,3 +109,22 @@ def validate_date_range(**kwargs):
     elif (('end_date' and 'start_date' in kwargs) and
             kwargs['end_date'] < kwargs['start_date']):
         raise ValueError('Earlier date should be lower than later date')
+
+
+def validate_room_labels(**kwargs):
+    """
+    Function to validate the room label string type
+    :params room_labels
+    """
+    room_labels = kwargs.get('room_labels')
+    new_label = re.search("[{}:]", str(room_labels))
```

Comment:
Why can't you just check that the type of variable is a list? What's the point in converting it to a string then looking for block brackets?

Reply:
The concern was that the mutation accepted a list of strings containing a dictionary. as can be seen in the screenshot. Therefore I had to use a regex to ensure the value of the string inside the dictionary is not invalid.

Reply:
Since the type of variable is a list I had to convert it into a string first before checking for block brackets.